packages/composites/bitcoin-por-indexer/README.md

```diff
@@ -0,0 +1,65 @@
+# Bitcoin Proof-of-Reserves Indexer Service Adapter
+
+The adapter provides an interface for retrieving total reserves for sets of addresses a Bitcoin PoR Indexer Service.
+
+## Configuration
+
+The adapter takes the following environment variables:
+
+| Required? |           Name            |             Description              | Options |       Defaults to       |
+| :-------: | :-----------------------: | :----------------------------------: | :-----: | :---------------------: |
+|           | `BITCOIN_POR_INDEXER_URL` | Bitcoin Por Indexer Service endpoint |         | `http://localhost:8080` |
```

Comment:
We should make this possible:

```suggestion
|           | `[network]_[chainId]_POR_INDEXER_URL` | Bitcoin Por Indexer Service endpoint |         | `http://localhost:8080` |
```

An example env var would be `BITCOIN_MAINNET_POR_INDEXER_URL`